streamer/src/nonblocking/quic.rs

```diff
@@ -79,6 +82,11 @@ const CONNECTION_CLOSE_REASON_TOO_MANY: &[u8] = b"too_many";
 /// Limit to 250K PPS
 pub const DEFAULT_MAX_STREAMS_PER_MS: u64 = 250;
 
+pub const DEFAULT_MAX_CONNECTIONS_PER_IPADDR_PER_MINUTE: u64 = 8;
+const TOTAL_CONNECTIONS_PER_SECOND: u64 = 2500;
+
+const CONNECITON_RATE_LIMITER_CLEANUP_THRESHOLD: usize = 100_000;
```

Comment:
This could probably be named better. It's hard to understand its use by its name.

Reply:
will rename